xla/pjrt/gpu/BUILD

```diff
@@ -116,6 +116,7 @@ cc_library(
         "@tsl//tsl/platform:status",
         "@tsl//tsl/platform:statusor",
         "@tsl//tsl/profiler/lib:connected_traceme",
+        "@tsl//tsl/profiler/lib:nvtx_utils",
```

Comment:
It seems adding this dependency unconditionally here causes a linker error if a target links in both nvtx_utils and nvtx_utils_libtpu. I could avoid this problem when I added this dependency inside if_cuda guard and guarded the include and the new code, but not sure this is the solution you would choose yourself.
We will revert this for now.